client/src/stories/spriteAnimation.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef } from "react";
+import { playerImage } from "./assets/spriteSheet";
+
+/*
+one image: 180 x 240 = srcWidth x srcHeight
+rows & cols = 4
+*/
+
+function drawPlayerSprite(player, context) {
+  const { width, height, left, top, direction } = player;
+
+  const srcWidth = 180;
+  const srcHeight = 240;
+  const startCol = 0;
+  let startRow;
+
+  switch (direction) {
+    case "FRONT":
+      startRow = 0;
+      break;
+    case "LEFT":
+      startRow = 1;
+      break;
+    case "RIGHT":
+      startRow = 2;
+      break;
+    case "JUMPING":
+      startRow = 3;
+      break;
+    default:
+      startRow = 0;
+      break;
+  }
+
+  const srcX = srcWidth * startCol;
+  const srcY = srcHeight * startRow;
+
+  context.beginPath();
+  context.drawImage(
+    playerImage,
+    srcX,
+    srcY,
+    srcWidth,
+    srcHeight,
+    left,
+    top,
+    width,
+    height
+  );
+  context.fillStyle = "#ff0167";
+  context.fill();
+  context.closePath();
+}
+
+function draw(context) {
+  context.fillStyle = "#000000";
+  context.fillRect(0, 0, context.canvas.width, context.canvas.height);
+}
+
+function rotatePlayer(keyCode, player) {
+  if (keyCode === "ArrowRight") {
+    player.direction = "RIGHT";
+  }
+  if (keyCode === "ArrowLeft") {
```

Comment:
Very small optimization:
```suggestion
  } else if (keyCode === "ArrowLeft") {
```

Right now, all the if-cases are checked, even if the first one is true.